hack/test-go.sh

```diff
@@ -218,7 +218,7 @@ for (( i=0, j=0; ; )); do
   apiVersion=${apiVersions[i]}
   etcdPrefix=${etcdPrefixes[j]}
   echo "Running tests for APIVersion: $apiVersion with etcdPrefix: $etcdPrefix"
-  KUBE_API_VERSION="${apiVersion}" KUBE_API_VERSIONS="v1" ETCD_PREFIX=${etcdPrefix} runTests "$@"
+  KUBE_API_VERSION="${apiVersion}" KUBE_API_VERSIONS="v1,experimental/v1alpha1" ETCD_PREFIX=${etcdPrefix} runTests "$@"
```

Comment:
KUBE_API_VERSION is used by package testapi to decide which version to test. Currently, it only allows us to specify one version. I plan to refactor it to accept multiple versions, more precisely, one version per group.

KUBE_API_VERSIONS specify which versions are exposed by the apiserver. It should be fine as long as KUBE_API_VERSIONS includes all versions in KUBE_API_VERSION.